enterprise/frontend/src/metabase-enterprise/clean_up/CleanupCollectionModal/utils.ts

```diff
@@ -0,0 +1,97 @@
+import dayjs from "dayjs";
+import { c, t } from "ttag";
+
+import type { StaleCollectionItem } from "metabase-types/api";
+
+export const itemKeyFn = (item: StaleCollectionItem) =>
+  `${item.id}:${item.model}`;
+
+export const dateFilterOptions = [
+  {
+    label: c(`Not used in over -1 day`).t`-1 day`,
+    value: "one-day-back",
+    duration: [[-1, "day"]],
```

Comment:
Why `-1 day` and not `1 day`?

Reply:
this is tomorrow, since it's the future it will always return every item as stale. this is/was just temporary for testing purposes. i've since removed it locally.